e2e/add-nx-to-monorepo/src/add-nx-to-monorepo.test.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  createNonNxProjectDirectory,
+  readProjectConfig,
+  runCommand,
+  tmpProjPath,
+  updateFile,
+} from '@nrwl/e2e/utils';
+import { Workspaces } from '@nrwl/tao/src/shared/workspace';
+
+describe('add-nx-to-monorepo', () => {
+  it('should not throw', () => {
```

Comment:
Should we want to check for anything else? e.g. that `@nrwl/workspace` was added to package.json, that `nx.json` is created etc.

Reply:
Yeah, these tests will be fleshed out more over time. Talking to @vsavkin it sounds like we will merge the add-nx repo into this one, at which point fleshing the e2e out for those packages will be much easier.